views/js/runner/provider/qti.js

```diff
@@ -656,7 +656,7 @@ define([
         unloadItem : function unloadItem(){
             var self = this;
 
-            self.trigger('disablenav disabletools');
+            self.trigger('beforeunloaditem disablenav disabletools');
```

Comment:
hmmm, should you not trigger the event unloaditem then use eventifier.before('unloaditem') instead ?
https://github.com/oat-sa/tao-core/blob/master/views/js/core/eventifier.js#L450-L457